thrust/system/tbb/detail/scan.inl

```diff
@@ -104,7 +104,12 @@ struct inclusive_body
 
   void reverse_join(inclusive_body& b)
   {
-    sum = binary_op(b.sum, sum);
+    // Only accumulate this functor's partial sum if this functor has been
+    // called at least once -- otherwise we'll over-count the initial value.
+    if (!first_call)
```

Comment:
Make sure the commit message makes a note of this fix so I'll pick it up in release notes.

Reply:
Done.